cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -1165,6 +1165,234 @@ int32_t gdv_fn_cast_intervalyear_utf8_int32(int64_t context_ptr, int64_t holder_
   auto* holder = reinterpret_cast<gandiva::IntervalYearsHolder*>(holder_ptr);
   return (*holder)(context, data, data_len, in1_validity, out_valid);
 }
+
+GDV_FORCE_INLINE
+gdv_int32 utf8_char_length(char c) {
```

Comment:
Check if that function already exists in that file, because I remember that some similar function is used in `upper` and `lower` functions

Reply:
Ok, I found it and remove the duplicity.